package.json

```diff
@@ -8,7 +8,8 @@
   "files": [
     "./dist/**/*"
   ],
-  "main": "./dist/ods-charts.js",
```

Comment:
The main file here should remain `dist/ods-charts.js` because this would be the default packaged version. The module version shouldn't be the default one.

For example, in Bootstrap:
```json
"main": "dist/js/bootstrap.js",
"module": "dist/js/bootstrap.esm.js",
```

Reply:
I test this solution (with the changes in webpack.config.js as well of course), and i got back the angular compilation errors :  
```
./src/app/graph/graph.component.ts:44:21-46 - Error: export 'getThemeManager' (imported as 'ODSCharts') was not found in 'ods-charts' (module has no exports)
./src/app/graph/graph.component.ts:45:14-43 - Error: export 'ODSChartsMode' (imported as 'ODSCharts') was not found in 'ods-charts' (module has no exports)
./src/app/graph/graph.component.ts:46:27-79 - Error: export 'ODSChartsCategoricalColorsSet' (imported as 'ODSCharts') was not found in 'ods-charts' (module has no exports)
./src/app/graph/graph.component.ts:47:24-78 - Error: export 'ODSChartsSequentialColorsSet' (imported as 'ODSCharts') was not found in 'ods-charts' (module has no exports)
./src/app/graph/graph.component.ts:48:19-54 - Error: export 'ODSChartsLineStyle' (imported as 'ODSCharts') was not found in 'ods-charts' (module has no exports)
```

NB : to update the angular projet before testing, I do
```
npm uninstall ods-charts
npm i ../..

```